File: diKoding/XamlMvvmBasic/XamlMvvmBasicVb/DataServiceInDesign.vb
Public NotInheritable Class DataServiceInDesign
    Implements IDataService

    Public Function GetPerson() As PersonData Implements IDataService.GetPerson
        Return New PersonData() With { _
            .Age = 38, _
            .FirstName = "James", _
            .LastName = "Bond" _
        }
    End Function
End Class

File: diKoding/XamlMvvmBasic/XamlMvvmBasicCs/DataServiceInDesign.cs
namespace XamlMvvmBasic
{
    public sealed class DataServiceInDesign
        : IDataService
    {
        public PersonData GetPerson()
        {
            return new PersonData
                {
                    Age       = 38, 
                    FirstName = @"James", 
                    LastName  = @"Bond"
                };
        }
    }
}